### ISSUE TITLE
[Web] [Accessibility] [11yMAS] [MAS2.4.3] [Focus is moving to Facebook link instead of moving to first interactive element(Office dev center logo) of the page.]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab-2017#674]
**User Experience:**
If focus order is not aligned with either the meaning or operation of the UI, it would be confusing for the Keyboard dependent users who navigate documents sequentially and expect the focus order to be consistent with the sequential reading order.

**Test Environment:**
OS Version: 16299.19
Browser: Microsoft Edge 41.16299.15.0

**Attachment**
[MAS2.4.3_Focus order.zip](https://github.com/OfficeDev/script-lab/files/1437473/MAS2.4.3_Focus.order.zip)
[MAS2_4_3_FocusOrder_MAC.zip](https://github.com/OfficeDev/script-lab/files/1456589/MAS2_4_3_FocusOrder_MAC.zip)

**Repro Steps:**
1.Open the [http://scriptlabdocsdemo.azurewebsites.net/Office Dev Center - Docs - Build your first Excel add-in.html](url)
2.Navigate to page using Tab key.
3.Verify focus is moving to Facebook link or not is incorrect. It should be Focus moves to facebook link and skips the headers.

**Current Behavior:**
Focus is moving to Facebook link instead of moving first interactive element(Office dev center logo) of the page.

**Expected Behavior:**
Focus should go to first interactive element which is Office dev center logo link instead of moving to Facebook link.

**Note:**
Same issue is repro for the [MAC + safari][ Build info: OS Build: Mac OS Sierra 0.12.6 Browser: Safari 11.0.1] Focus order is not correct as user navigate from the top of the page, focus moves to Add this sharing controls instead of navigating to first interactive element on the page.

**MAS Reference:**
[https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7b0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9%7d](url)

**Suggested Fix:**
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=2.4.3](url)
